### PR TITLE
feat: add support for tomtom go fleet & Sygic Truck

### PR DESCRIPTION
**Pull request by amrahmed242**
## What the PR Includes
- [x] Contributes to https://github.com/mattermoran/map_launcher/issues/111
- [x]  Add support for the TomTom Go Fleet on Android (released only for android)
- [x]  Add support for Sygic navigation app on Android and iOS.

### Limitations
The SVG image is a converted PNG icon, and has a relatively low quality.
No titles or other parameters are supported for Sygic.
> note: sygic document mention that we can pass destination title but did't work.

### Tesing
The marker and directions command were both verified to work on the following devices.

- Google pixel 5
- iphone x

### CODE REVIEW

**Comment by amrahmed242** [comment]
@mattermoran 
Couldn’t assign you for review so i tagged you in this comment, thanks.

File: lib/src/directions_url.dart [hunk 1]
```diff
@@ -256,5 +256,29 @@ String getMapDirectionsUrl({
...
+        url:
+            'com.sygic.aura://coordinate|${destination.longitude}|${destination.latitude}|drive',
+        queryParams: {
+          ...(extraParams ?? {}),
+        },
+      );
+      print(url);
+      print(url);
+      print(url);
+      print(url);
```
Comment (frankvollebregt) [inline lib/src/directions_url.dart:281]: This should probably be removed before merging this into the package

**Review by frankvollebregt — CHANGES_REQUESTED** [review]
Perhaps you should give the `tomtomgofleet.svg` file a solid white background, so that it won't be near invisible on a dark screen.
Should be easy using something like:
```xml
<rect width="480" height="480" fill="#fff"/>
```

Additionally, on my device (Google Pixel 7), the Sygic app does not show up in the available maps even though it is installed. I had a similar issue with TomTom Go before, but that shows up since this PR, so perhaps something else is going on under the hood...

**Comment by mattermoran** [comment]
@amrahmed242 Thanks for contribution!
This will be available in the next release.

**Comment by frankvollebregt** [comment]
It seems like these two apps weren't added to the README file yet in the master branch, so perhaps it'd be good to do that in additon to the changes I proposed above @mattermoran 

I'll try to do it myself and make a PR in the next hour or two

**Comment by mattermoran** [comment]
Released now in v3 :)

Sygic non truck will have to come later otherwise it would've kept dragging

**Pull request merged**